HLTrigger/Configuration/python/customizeHLTforPatatrack.py

```diff
@@ -230,9 +225,6 @@ def customisePixelLocalReconstruction(process):
     # workaround for AlCa paths
 
     if 'AlCa_LumiPixelsCounts_Random_v1' in process.__dict__:
-        if "HLTSchedule" in process.__dict__:
-            ind = process.HLTSchedule.index(process.AlCa_LumiPixelsCounts_Random_v1)
-            process.HLTSchedule.remove(process.AlCa_LumiPixelsCounts_Random_v1)
```

Comment:
@missirol these adding/removing of the paths to/from the schedule are no longer necessary, because the "official" schedule object handles them automatically ?

Reply:
Yes, that's my understanding from Matti's explanations (and I verified it by hand on a few examples). Since in this PR I used `customizeHLTforAll` to do the trick, that should always be used before other customisations, and afaik that's always the case.